src/util.js

```diff
@@ -43,7 +43,8 @@ export function stringify (data) {
   return CircularJSON.stringify(data, replacer)
 }
 
-function replacer (key, val) {
+function replacer (key) {
+  const val = this[key]
```

Comment:
@yyx990803 This is to prevent the default replacer to turn Dates into Strings before calling the custom replacer. See https://github.com/WebReflection/circular-json/issues/4